### PR TITLE
Feliciana:Vents fixed

### PR DESCRIPTION
**Pull request by Altangy**
:cl:
add: HoS Hardsuit
add: Hydroponics CondiMaster and Grinder
add: Bar atmos alarm in the bar section and Condimaster
fix: 24 Vents somehow were set to be part of an engine. This has been corrected.
experiment: Courtroom Intercom range was increased for court cases. Let me know how this works out
/:cl:

[why]: # (Please add a short description [on the next line] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding:)

### CODE REVIEW

File: _maps/map_files/Feliciana/Feliciana.dmm [hunk 1]
```diff
@@ -2829,7 +2829,7 @@
 "bcu" = (/obj/machinery/door/poddoor{id = "weapon_k_1"; name = "munitions launcher bay door"},/obj/structure/fans/tiny,/obj/structure/fans/tiny,/obj/effect/turf_decal/stripes/line{dir = 1},/turf/open/floor/engine,/area/shuttle/ftl/munitions/loading)
 "bcv" = (/obj/structure/chair{dir = 4},/obj/machinery/airalarm{dir = 4; icon_state = "alarm0"; pixel_x = -22},/turf/open/floor/plasteel,/area/shuttle/ftl/maintenance/bridge)
 "bcw" = (/obj/structure/table/wood,/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden,/turf/open/floor/plasteel,/area/shuttle/ftl/maintenance/bridge)
-"bcx" = (/obj/structure/table/wood,/obj/item/device/radio/intercom,/turf/open/floor/plasteel,/area/shuttle/ftl/maintenance/bridge)
+"bcx" = (/obj/structure/table/wood,/obj/item/device/radio/intercom{canhear_range = 5},/turf/open/floor/plasteel,/area/shuttle/ftl/maintenance/bridge)
```
Comment (Vivalas) [inline _maps/map_files/Feliciana/Feliciana.dmm]: Is this default for intercoms?

**Comment by Vivalas** [comment]
Looks good, would prefer pictures, leaving open for @Qustinnus and @IndusRobot to look at.

File: _maps/map_files/Feliciana/Feliciana.dmm (hunk 1 above)
Comment (Vivalas) [reply _maps/map_files/Feliciana/Feliciana.dmm]: Nevermind read the diff

**Comment by Qustinnus** [comment]
what viv said, post pics in future.

although these changes are not too big and not very large scale so it should be fine

**Pull request merged**